packages/calypso-build/webpack/jsx.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Return a webpack loader object containing our jsx (jsx -> js) stack.
+ *
+ * @param  {Object}    _                    Options
+ * @param  {integer}   _.workerCount        Number of workers that are being used by the thread-loader
+ * @param  {string}    _.configFile         Babel loader config file
+ * @param  {string}    _.cacheDirectory     Babel loader cache directory
+ * @param  {string}    _.cacheIdentifier    Babel loader cache identifier
+ *
+ *
+ * @return {Object}   webpack loader object
+ */
+module.exports.loader = ( { workerCount, configFile, cacheDirectory, cacheIdentifier } ) => ( {
```

Comment:
It would be good to throw some errors if we receive bad params or are missing params that we require.

What do you think?

Reply:
Sure we can check if files or directories exits and if not then provide some better defaults or throw errors. 
Also where should we be documenting the shards?